src/app.html

```diff
@@ -3,10 +3,18 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.jpg" />
+		<link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
It is for importing the typography used in the design

Reply:
we use fontsource to actually import typograpy, i just saw that the font Inter in fontsource website. But since this in an html file i don't know which is the best approach
<img width="366" alt="image" src="https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/46734336/34d8031e-0328-4438-9a14-19bc0fd1693e">

Reply:
@AdderlyPM  , can we use fontsource for this type as well?